database/authorPG.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const faker = require('faker');
+const { Pool, Client } = require('pg');
+
+const host = process.env.PG_HOST;
+const port = process.env.PG_PORT;
+const username = process.env.PG_USER;
+const password = process.env.PG_PASS;
+const database = process.env.PG_DB;
+
+const pool = new Pool({
+    user: username,
+    host: host,
+    database: database,
+    password: password,
+    port: port,
+});
+
+pool.connect();
+
+function seed() {
+    let insertSql, values;
+    let author_id, first_name, middle_name, last_name, job, employer = '', rating, reviews, students, courses, thumbnail, bio;
```

Comment:
I think you can remove line 23 and do let author_id = i ......instead.  I am unclear whether that will help you save some memory leakage though.

Reply:
I will take a look at this.